conformance/1.7/Makefile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TEST_PROFILE ?= kf-conformance-test
+KUBEFLOW_NAMESPACE ?= kubeflow
+
+setup:
```

Comment:
NIT: Replace here and below:
```
.PHONY: setup
setup:
...
```

This will prevent the `make <rule>` from failing when there is a `<rule>` file inside the folder. For example, if someone removes the `.yaml` extension from `setup.yaml`, `make setup` won't behave as expected.

Reply:
Done